### PR TITLE
Attempt to fix job listing syncing

### PR DESCRIPTION
**Pull request by jorunfa**

### CODE REVIEW

**Review by jonny-wg2 — APPROVED** [review]
Looks good! We might need to create/add a GITHUB_TOKEN in the secrets. Let's merge to test and see. https://github.com/working-group-two/wgtwo.com/pull/94/files#diff-c1eeb8a35dd0a10cef85294380fb15f0e863489bd8290abb2e97c7ced1c2e973R28

File: .github/workflows/website-sync-personio-jobs.yml [hunk 1]
```diff
@@ -11,16 +11,20 @@ jobs:
...
+        id: cpr
+        uses: peter-evans/create-pull-request@v4
+        with:
+          delete-branch: true
+          commit-message: "[autosync] Update personio job list"
+      - name: Enable Automerge on PR
+        if: steps.cpr.outputs.pull-request-operation == 'created'
+        uses: peter-evans/enable-pull-request-automerge@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (jonny-wg2) [inline .github/workflows/website-sync-personio-jobs.yml:28]: We might need to create this GITHUB token?

Comment (jorunfa) [reply .github/workflows/website-sync-personio-jobs.yml:28]: > Let's merge to test and see.

a-ok 👍

Comment (jonny-wg2) [reply .github/workflows/website-sync-personio-jobs.yml:28]: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#about-the-github_token-secret

Should be good. This token is auto generated

**Pull request merged**